src/stars.css

```diff
@@ -0,0 +1,36 @@
+.stars {
+  display: flex;
+}
+
+.stars > .stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url("./images/star-active.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--0 > .stars__star:nth-child(n+1) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--1 > .stars__star:nth-child(n+2) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--2 > .stars__star:nth-child(n+3) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--3 > .stars__star:nth-child(n+4) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--4 > .stars__star:nth-child(n+5) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--5 > .stars__star:nth-child(n+6) {
+  background-image: url("./images/star.svg");
+}
```

Comment:
It is a good example when you actually can group selector because they MUST have the same styling and one can't change without the others